x-pack/packages/ml/anomaly_utils/aggregation_types.ts

```diff
@@ -101,9 +121,11 @@ function getBasicMlOnlyAggregation(): Omit<Aggregation, 'id' | 'title' | 'fields
   };
 }
 
-// list of aggregations only support by ML and which don't have an equivalent ES aggregation
-// note, not all aggs have a field list. Some aggs cannot be used with a field.
-export const mlOnlyAggregations: Aggregation[] = [
+/**
+ * List of aggregations only support by ML and which don't have an equivalent ES aggregation.
```

Comment:
nit: 'only supported by ML'

Reply:
Fixed in [7a447f5](https://github.com/elastic/kibana/pull/155530/commits/7a447f56a22c6bf1ce76549677e6b375786b518f).